testing-guidance/workflow-tests/Viewport/viewport-workflow-tests.md

```diff
@@ -0,0 +1,74 @@
+1. Console log errors/warnings/spam
+2. Asserts
+​
+### Platforms:​
+- Windows
+- Linux​
+​
+### Documentation pages:​
+* [Working with the 3D Viewport](https://www.o3de.org/docs/user-guide/editor/Viewport/)
+* [Reference Spaces](https://www.o3de.org/docs/user-guide/editor/Viewport/reference-spaces/)
+* [Group Selection](https://www.o3de.org/docs/user-guide/editor/Viewport/group-selection/)
+* [Advanced 3D Viewport Operations](https://www.o3de.org/docs/user-guide/editor/Viewport/advanced-operations/)​
+​
+### Workflow 1: Camera control 
+​
+**Product:** 
+A Level with multiple entities with a mesh component and assigned shaderball model asset.
+​
+**Suggested Time Box:** 5-10 mins
+
+| Workflow                                                               | Requests																																																																						   | Things to Watch For                                                                                                                                                          |
+|------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Navigate level with camera controls									 | <ol><li>Select the Viewport.</li><li>Navigate the camera with W, A, S, D, Q, E. Hold shift for fast move.</li><li>Right-click and drag to rotate the camera.</li><li>Press MMB and drag to pan up and down.</li><li>Scroll up and down using the MMB, to zoom in and out.</li><li>ALT+LMB to orbit.</li></ol>| <ul><li>Correct camera behaviour.</li></ul>																													  |
+| Be this camera option													 | <ol><li>Select the camera entity or create one.</li><li>In the Entity Inspector, press the "Be this camera" button.</li><li>Move the camera around.</li><li>Press the Return to default editor camera button.</li></ol>																		   | <ul><li>Correct camera behaviour.</li></ul>																																  |
+| Viewport camera selector												 | <ol><li>Open the Viewport Camera Selector (Tools > Viewport > Viewport Camera Selector).</li><li>Right-Click in the Viewport and select "Create camera entity from view".</li><li>Navigate to a different location in the Viewport and create another camera.</li><li>Navigate through the available cameras in the Viewport Camera Selector while monitoring the Viewport.</li></ol>| <ul><li>Cameras added to the level.</li><li>Correct camera switching and behaviour.</li></ul>|
+---
+
+### Workflow 2: Viewport options  
```

Comment:
It would be good to have an explicit section about the viewport title dialog (top right of viewport) to sanity check those options work correctly (e.g. FOV and show/hide helpers/icons etc...)